sentry-android-core/src/test/java/io/sentry/android/core/ScreenshotEventProcessorTest.kt

```diff
@@ -85,16 +78,27 @@ class ScreenshotEventProcessorTest {
 
     @Test
     fun `when close is called and  attach screenshot is disabled, does not unregisterActivityLifecycleCallbacks`() {
-        val sut = fixture.getSut(false)
+        val sut = fixture.getSut()
 
         sut.close()
 
         verify(fixture.application, never()).unregisterActivityLifecycleCallbacks(any())
     }
 
+    @Test
+    fun `when process is called and attachScreenshot is disabled, unregisterActivityLifecycleCallbacks`() {
+        val sut = fixture.getSut()
+        val hint = Hint()
+
+        val event = fixture.mainProcessor.process(getEvent(), hint)
+        sut.process(event, hint)
+
+        verify(fixture.application).registerActivityLifecycleCallbacks(any())
```

Comment:
Should this be `unregisterActivityLifecycleCallbacks` instead?

Reply:
Good catch